### DISCUSSION TITLE
How can you remove the top right navigation element?

### DISCUSSION THREAD

**Discussion by rpmcoding** [Budibase/budibase#7931] [Help] [answered]
I need to remove this element on the top right of the navigation that links to the builder, any ideas how?

![screenshot](https://user-images.githubusercontent.com/59533498/191943953-6fddc0c3-52e4-49bf-8bba-c6f9433aebce.png)

**Comment by melohagan** [comment, marked as answer]
Hey @rpmcoding 

These may help:
 - https://github.com/Budibase/budibase/discussions/1941 (Recommends that you don't remove it)
 - https://github.com/Budibase/budibase/discussions/6522 (How to do it)

**Reply by rpmcoding** [reply]
Thanks @melohagan it would be great to have a toggle on/off for it, but the embed hack is good for the meantime.

I am working on a public facing app and the users don't need access to builder nor do I want them to go to it.  Hopefully that gives some context.
